### ISSUE TITLE
Create a landing page

### ISSUE THREAD

**Issue by ziggysauce** [jduong09/water-your-subs#10]
Blocked by https://github.com/jduong09/gitcoding/issues/1 (react boilerplate)
Blocked by https://github.com/jduong09/gitcoding/issues/2 (node/express server)

### Backend
* Create a GET route for the landing page

### Frontend
* Create a landing page UI
* This will eventually require(?) authentication before proceeding

**Comment by jduong09** [comment]
The landing page will include a sign-in button, which will route the user to a login page correct? And then after logging in, the user will be directed to the app with their content correct?

**Comment by ziggysauce** [comment]
Yah, I think that's the move. At least for now. I think _eventually_ we would want more of a landing page with info about the app to entice the user to want to use it/create a profile. but for now, that simple call-to-action button should suffice

**Comment by ziggysauce** [comment]
Things to do for this ticket:
- [x] When navigating to the base `'/'` route, show a sign-in button if the user has not logged in yet
- [x] After logging in, show the "homepage" (this is where we'll add all our content)
  - [ ] Display a logout button (should only show this button when a user is already logged in)
- [x] Clean up all that boilerplate create-react-app code (we can get rid of it now)
- [x] Comment out or remove the "click me" and "create user" buttons. We'll want to create a separate ticket with database schema functionality to better fit out needs
- [x] Get rid of that fontawesome coffee icon and instead use a different icon for both the login and logout buttons
